xDrip/ApplicationSupport/AppDelegate.swift

```diff
@@ -7,21 +7,64 @@
 //
 
 import UIKit
+import UserNotifications
+#if !targetEnvironment(macCatalyst)
+import AVKit
+#endif
 
 // swiftlint:disable discouraged_optional_collection
-
 @UIApplicationMain final class AppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
     
+    #if !targetEnvironment(macCatalyst)
```

Comment:
This definitely doesn't belong to the `AppDelegate`. Create a separate worker and instantiate it in `ApplicationLaunchController`